package.json

```
@@ -68,6 +68,43 @@
       }
     ],
     "languageModelTools": [
+            {
+        "name": "list_Topics",
```

Minor suggestion just for consistent formatting with the others
```suggestion
        "name": "list_topics",
```